client/app/bundles/course/forum/components/buttons/HideButton.tsx

```diff
@@ -18,6 +18,14 @@ const translations = defineMessages({
     id: 'course.forum.HideButton.unhide',
     defaultMessage: 'Unhide',
   },
+  hideTag: {
```

Comment:
What about translation files?

Reply:
updated.
Also, I have updated the mandarin one, though you might need to double check if my translation is correct already or not